packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -26,11 +27,13 @@ export interface MenuButtonProps extends IButtonProps {
   menuItems?: MenuButtonItemProps[];
   onItemClick?: (key: string) => void;
   contextualMenu?: ContextualMenuProps;
+  primary?: boolean;
 }
 
 export type MenuButtonSlotProps = {
   root: MenuButtonProps;
   button: IButtonProps & MenuButtonTokens;
+  primaryButton: IButtonProps & MenuButtonTokens;
   contextualMenu: React.PropsWithRef<ContextualMenuProps>;
   contextualMenuItems: Pick<MenuButtonProps, 'menuItems'>;
```

Comment:
sidebar even though it's not part of this PR: what are the contextualMenuItems and contextuMenuItem slots providing that can't be handled by the contextualMenu slot alone?  Props I would have expected to propagate up/down through contextualMenu, but were there additional needs / scenarios supported that require these slots?

Reply:
ContextualMenu has children prop. Props are not propagated from parent component.

Reply:
I don't think I'm articulating this well, I need more time in the framework :( I'm interested in why we need a slot for contextualMenuItems, that feels like an implementation detail of contextualMenu -- for example if the ContextualMenuProps had a simple menuStrings[] property and that was the only way to generate menu items, I wouldn't expect this slot.  That isn't how the contextualMenu is built so it's somewhat irrelevant, but I'm not sure how to interpret what the contextualMenuItems slot is providing in this instance -- will any component with a ContextualMenu slot also need to have a ContextualMenuItems slot?

Reply:
I think yes. @lenahong please confirm. Originally ContextualMenu was planned not to have items passed in as IContextualMenuItem[]. It was planned to have menu items as children